File: Plugin/app/src/main/java/com/qihoo360/replugin/sample/demo1/ThemeBlackNoTitleBarActivity.java
package com.qihoo360.replugin.sample.demo1;

import android.app.Activity;
import android.os.Bundle;
import android.view.Gravity;
import android.widget.FrameLayout;
import android.widget.LinearLayout;
import android.widget.TextView;

import androidx.annotation.Nullable;

import static android.view.ViewGroup.LayoutParams.MATCH_PARENT;

/**
 * @author liuyang 2023/4/6 8:25 PM
 */
public class ThemeBlackNoTitleBarActivity extends Activity {

    TextView textView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        LinearLayout root = new LinearLayout(this);
        root.setPadding(30, 30, 30, 30);
        FrameLayout.LayoutParams lp = new FrameLayout.LayoutParams(MATCH_PARENT, MATCH_PARENT);
        root.setLayoutParams(lp);

        setContentView(root);

        // textView
        textView = new TextView(this);
        textView.setGravity(Gravity.CENTER);
        textView.setTextSize(30);
        textView.setText("Theme: BlackNoTitleBar");
        LinearLayout.LayoutParams lp2 = new LinearLayout.LayoutParams(MATCH_PARENT, MATCH_PARENT);
        textView.setLayoutParams(lp2);
        root.addView(textView);
    }
}

File: Plugin3/app/src/main/java/com/qihoo360/replugin/sample/demo3/activity/theme/ThemeBlackNoTitleBarActivity.kt
package com.qihoo360.replugin.sample.demo3.activity.theme

import android.app.Activity
import android.os.Bundle
import android.view.Gravity
import android.view.ViewGroup
import android.widget.FrameLayout
import android.widget.LinearLayout
import android.widget.TextView

class ThemeBlackNoTitleBarActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val root = LinearLayout(this)
        root.setPadding(30, 30, 30, 30)
        val lp = FrameLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT)
        root.layoutParams = lp

        setContentView(root)

        // textView
        val textView = TextView(this)
        textView.gravity = Gravity.CENTER
        textView.textSize = 30f
        textView.text = "Theme3: BlackNoTitleBar"
        val lp2 = LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.MATCH_PARENT)
        textView.layoutParams = lp2
        root.addView(textView)
    }
}